### ISSUE TITLE
When installed, nothing works, `winget.exe` file is actually empty.

### ISSUE THREAD

**Issue by Ayfri** [microsoft/winget-cli#1022]
# Brief description of your issue
Hi, I tried to install winget through the Microsoft Store, but what I get is a CLI that doesn't work and when I search for the file `winget.exe` in `%LOCALAPPDATA%\Microsoft\WindowsApps\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\winget.exe`, the file is `0ko`.
![image](https://user-images.githubusercontent.com/34159136/120088733-01d19e00-c0f4-11eb-9cf1-d54edc76fefb.png)

# Steps to reproduce
Install winget, try launching any command.

# Expected behavior
Getting winget to work.

# Actual behavior
Getting nothing when executing any winget command.

# Environment

```none
Windows: 20h1 10.0.19041.964
Package: 1.0.11451
```

**Comment by vasu2912** [comment]
Seems to be having the same issue here. Tried all the ways still unable to get to run it.

![image](https://user-images.githubusercontent.com/30508264/120091187-69a0dc80-c126-11eb-9a65-989c804e0f43.png)

**Comment by SirObby** [comment]
![image](https://user-images.githubusercontent.com/70753342/120093388-981cb880-c122-11eb-8b1a-51c5c791c4df.png)
Hopefully this answers the issue.

**Comment by sumanth-mandalapu** [comment]
Hi, I faced the same issue , could not launch app from Microsoft store and winget is not recognized. I followed the steps below:
1.Turned on Developer mode . Still did not work.
2.Downloaded from  the link mentioned in [ThioJoe](https://www.youtube.com/watch?v=uxr7m8wDeGA  ) winget video description.Here is the link :  ms-appinstaller:?source=https://aka.ms/getwinget . Copy paste it in browser.
3.It prompted to launch AppInstaller as it is already installed from Store. Updated the AppInstaller.
4.Now launched it from Mircosoft store.
5.winget is working in CMD.
